### ISSUE TITLE
Is the --init-weights option correct?

### ISSUE THREAD

**Issue by sdgroeve** [percolator/percolator#292]
Hi,

I saved the SVM weights in one Percolator run with the --weights option.
Then I run again with the --init-weights option pointing to the saved weights.
What happens then? It seems just the first SVM weights (from the 3CV) is used to score all PSMs.
Is that correct?

Thnx

**Comment by percolator** [comment]
Yes, percolator selects one of the three CV directions as the init direction. We set it up so that it always selects the first one.
